Patch by schwardt@univention.de (Wed Sep 11 16:07:55 2013 +0200):

use debian library path for gnu-efi



--- shim-0.2.ORIG/Makefile
+++ shim-0.2/Makefile
@@ -6,7 +6,7 @@ LIB_PATH	= /usr/lib64
 
 EFI_INCLUDE	= /usr/include/efi
 EFI_INCLUDES	= -nostdinc -ICryptlib -ICryptlib/Include -I$(EFI_INCLUDE) -I$(EFI_INCLUDE)/$(ARCH) -I$(EFI_INCLUDE)/protocol -Iinclude
-EFI_PATH	:= /usr/lib64/gnuefi
+EFI_PATH	:= /usr/lib
 
 LIB_GCC		= $(shell $(CC) -print-libgcc-file-name)
 EFI_LIBS	= -lefi -lgnuefi --start-group Cryptlib/libcryptlib.a Cryptlib/OpenSSL/libopenssl.a --end-group $(LIB_GCC) 
